client/src/components/Q&A/SearchBar.jsx

```diff
@@ -1,17 +1,27 @@
 /* eslint-disable import/extensions */
 import React from 'react';
 
-export default function SearchBar() {
+export default function SearchBar({ setSearchTerm }) {
+  function handleSearch(event) {
+    const search = event.nativeEvent.target.value;
+    if (search.length >= 3) {
+      setSearchTerm(search);
+    } else if (search.length === 0) {
+      setSearchTerm('');
+    }
+  }
+
   return (
     <div
-      className="searchBar"
+      className="search-bar"
       style={{ border: '2px solid slateblue' }}
     >
       Hello from Search Bar
       <form>
         <input
-          className="searchInput"
+          className="search-input"
           placeholder="Have a question? Search for answers…"
```

Comment:
Does the search bar look for answers or questions? Or both?

Reply:
I think it filters the questions, but I could be mistaken too. In the business req's it doesn't mention searching by answers.